modules/kubernetes-addons/aws-privateca-issuer/locals.tf

```diff
@@ -0,0 +1,46 @@
+locals {
+  name                 = "aws-privateca-issuer"
+  service_account_name = "${local.name}-sa"
+
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://cert-manager.github.io/aws-privateca-issuer"
+    version     = "0.1.2"
+    namespace   = local.name
+    description = "AWS PCA Issuer helm Chart deployment configuration."
+    values      = local.default_helm_values
+    timeout     = "1200"
+  }
+
+  default_helm_values = []
+
+  helm_config = merge(
+    local.default_helm_config,
+    var.helm_config
+  )
+
+  set_values = [
+    {
+      name  = "serviceAccount.create"
+      value = "false"
+    },
+    {
+      name  = "serviceAccount.name"
+      value = local.service_account_name
+    }
+  ]
+
+  irsa_config = {
+    create_kubernetes_namespace       = true
+    kubernetes_namespace              = local.name
+    create_kubernetes_service_account = true
+    kubernetes_service_account        = local.service_account_name
+    irsa_iam_policies                 = [aws_iam_policy.aws_privateca_issuer.arn]
```

Comment:
We need to provide options to add additional IAM policies to this IAM role
```hcl
irsa_iam_policies                 = concat([aws_iam_policy.aws_privateca_issuer.arn], var.irsa_policies)
```

and add the following to `varriables.tf`

```hcl
variable "irsa_policies" {
  type        = list(string)
  description = "Additional IAM policies for a IAM role for service accounts"
  default     = []
}
```